src/room.py

```diff
@@ -2,7 +2,8 @@
 # description attributes.
 
 class Room:
-    def __init__(self, name, description):
+    def __init__(self, name, description, list):
         #attributes
         self.name = name
-        self.description = description
\ No newline at end of file
+        self.description = description
+        self.list = list
```

Comment:
I will not use the word list here as a class property name because it is a default keyword method in Python to convert an iterable object into a list.